src/huak/ops/install.rs

```diff
@@ -58,7 +58,7 @@ pub mod tests {
 
     // TODO
     #[test]
-    fn installs_dependenices() {
+    fn installs_dependencies() {
```

Comment:
ty :) I'll have more time soon, so hopefully less of these in the future. Sorry!

Reply:
Ha, my IDE just points typos out and they're too easy to pass up. Keep em' coming (maybe not intentionally though ha)

Reply:
Yeaaa i should prob use an extension. lol